dask/dataframe/io/parquet.py

```diff
@@ -0,0 +1,78 @@
+import struct
+from dask.bytes.core import open_files
+
+try:
+    import fastparquet
+    from fastparquet import parquet_thrift
+except:
+    fastparquet = False
+
+
+def parquet_to_dask_dataframe(url, columns=None, filters=[],
+                              categories=None, **kwargs):
+    if fastparquet is False:
+        raise ImportError("fastparquet not installed")
+    import dask.dataframe as dd
+    from dask import delayed
+    try:
+        f = open_files(url+'/_metadata')[0].compute()
```

Comment:
Instead, should separate out the parse-and-open code, and the delayed version of it - this should call the direct one, unless I were to make another delayed function depending on the outputs of the make_part_file calls below.